bbc_sale/views/product.xml

```diff
@@ -122,6 +122,15 @@
                 <field name="public_categ_ids" position="before">
                     <field name="is_synced_magento"/>
                 </field>
+                <xpath expr="//group[@name='sale_condition']/div" position="attributes">
```

Comment:
Other modules add divs as well so you might want to be a little bit more specific by selecting on the div that contains the warranty field: `expr="//group[@name='sale_condition']/div/field[@name='warranty']/.."`

You assign a name to the div first, and then select it again by its name. I think you can skip the first step.